### DISCUSSION TITLE
How to increase time for flash message in symfony.

### DISCUSSION THREAD

**Discussion by papnoisanjeev** [symfony/symfony#44322] [Q&A]
How to increase flash message time in symfony.
As currently time is too short need to increase it.

**Comment by alessandro-podo** [comment]
flash messages only appears one time. 
What you mean with increase the time?

**Comment by papnoisanjeev** [comment]
@alessandro-podo 
Actually flash popup shows for a short duration if in any case text written is long then user not able to read it properly.
so we need it for at least 15-20 sec.

**Comment by youssefsaoubou** [comment]
Maybe you can use https://php-flasher.github.io/ to custom your flasher (time, ....)

**Comment by lyrixx** [comment]
Flash message are stored in the session. You may have something in you application that displays them. You need to search in **your** codebase to increase the duration
